packages/@react-aria/landmark/src/useLandmark.ts

```diff
@@ -391,26 +391,43 @@ class LandmarkManager implements LandmarkManagerApi {
   }
 
   public createLandmarkController(): LandmarkController {
-    let instance = this;
+    let instance: LandmarkManager | null = this;
     instance.refCount++;
     instance.setupIfNeeded();
     return {
       navigate(direction, opts) {
-        return instance.navigate(opts?.from || document.activeElement, direction === 'backward');
+        let element = opts?.from || document?.activeElement;
+        if (element && instance) {
```

Comment:
I do think throwing if `instance` is null would be appropriate here since you can't use a controller after it is disposed. And seems reasonable to assert that document is defined when using these APIs as well rather than silently doing nothing. You could do `instance!.navigate(document!.activeElement, ...)` etc